commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -215,6 +215,10 @@ def ask_aws_for_instances(env_name, aws_config, count):
             subprocess.call(['commcare-cloud-bootstrap', 'terminate',  env_name])
             print("Deleting file: {}".format(cache_file))
             os.remove(cache_file)
+        env_dir = get_environment(env_name).paths.get_env_file_path('')
+        if os.path.isdir(env_dir):
+            print("Deleting environment dir: {}".format(env_name))
```

Comment:
I think it'd be courteous to add to the warning that it will delete the environment

Reply:
https://github.com/dimagi/commcare-cloud/pull/1996/commits/d7ddee1abeab216d969c1605a991a9a431c84a87